CHANGELOG.md

```diff
@@ -47,6 +47,7 @@ Every module contains its own CHANGELOG.md. Please refer to the module you are i
 ### Improvements
 
 * (client) [#21436](https://github.com/cosmos/cosmos-sdk/pull/21436) Use `address.Codec` from client.Context in `tx.Sign`.
+* (cosmovisor) [#21487](https://github.com/cosmos/cosmos-sdk/pull/21487) Fix typo daemon
```

Comment:
Hey we don't need a changelog for this.

Reply:
fixing test case, will update changelog with new information